src/main.cc

```diff
@@ -178,6 +178,25 @@ signal_cb(ev_loop *loop, struct ev_signal *w, int revents)
 	tarantool_exit(0);
 }
 
+static void
+signal_int_cb(ev_loop *loop, struct ev_signal *w, int revents)
```

Comment:
Re name: signal_**sig**winch_cb() is right below, pelase name this callback consistently.

Reply:
Done.